scripts/4_transfer_balance.js

```diff
@@ -0,0 +1,39 @@
+require("dotenv").config();
+const hre = require("hardhat");
+const ethers = hre.ethers;
+const { assert } = require("chai");
+
+const { CREDENTIALS_ADDRESS } = process.env;
+
+async function main() {
```

Comment:
@gr8h what this script is doing?

Reply:
@leonprou  removed was just temporary added to transfer some funds